TypeInjections/Tool/Tool.fs

```diff
@@ -36,15 +39,24 @@ module Tool =
           [<Option('c', "clone", Required = false, HelpText = "input a git URL")>]
           url: string
           [<Option('f', "force", Required = false, HelpText = "overwrite the Old and New folders in the Output folder")>]
-          force: bool }
+          force: bool
+          [<Option('i', "ignore", Required = false, HelpText = "specify filename(s) you want to ignore")>]
+          ignore: seq<string>}
+        
+    (* TypecartProject's 2nd argument can be:
+    1. the absolute path to .typecartignore file
+    2. a list of regex values *)
+    let handleIgnore(ignores: seq<string> ) : Regex list =
+        let li = ignores |> Seq.toList
+        List.map (fun (x:string)  -> x |> Regex) li 
```

Comment:
This is unecessary. The `Typecart` API can already parse the ignores file. You should just supply the file name directly to the API.

Reply:
This is not me trying to parse a .typecartignore file, I wanted the user to just write out the strings to ignore in the command
Ex. 
`$ typecart git -o s232s -n fd2323 --ignore file1 file4 utils`
This function is just casting this string input from the user into a Regex string typeCart can use. It seems a little easier, and the API provided a `Regex list` argument for the `typecartProject` constructor

Reply:
See my comment below. You should supply a list of strings to the API.

Reply:
Yeah I can't find any typecartProject constructor that allows a `string list`

Reply:
For now, we'll just support path to .typecartignore